Sources/Danger/Plugins/SwiftLint/SwiftLint.swift

```diff
@@ -91,7 +94,10 @@ extension SwiftLint {
         violations = violations.map { violation in
             let updatedPath = violation.file.deletingPrefix(currentPath).deletingPrefix("/")
             var violation = violation
-            violation.update(file: updatedPath)
+            violation.file = updatedPath
+            if strict {
+                violation.severity = .error
```

Comment:
can you please add a specific test for this? This would (I think), also change how the violation gets displayed on the markdown table, in that case that should be tested as well 👍